lib/matplotlib/widgets.py

```diff
@@ -2334,6 +2334,12 @@ def _set_active_handle(self, event):
 
     @property
     def geometry(self):
+      """
```

Comment:
Sorry I missed this before, but the whole string will need to be indented by 4 spaces (instead of the current 2) to line up with the if statement below.